rotkehlchen/api/v1/schemas.py

```diff
@@ -1828,47 +1830,38 @@ def make_assets_post_query(
         return {'filter_query': filter_query, 'identifiers': data['identifiers']}
 
 
-class AssetsSearchLevenshteinSchema(DBOrderBySchema, DBPaginationSchema):
+class AssetsSearchLevenshteinSchema(Schema):
```

Comment:
No, I don't think so since "pagination" happens only at python level, nothing at sqlite level and `offset` was never used, only `limit`. https://github.com/rotki/rotki/blob/develop/rotkehlchen/globaldb/handler.py#L461 and order-by rules were not making much sense since all the values are later sorted in python anyway https://github.com/rotki/rotki/blob/develop/rotkehlchen/globaldb/handler.py#L461

Reply:
Hmm okay that's only temporary then. This will go away as a method at some point. Pagination should not happen at python.